invenio_circulation/api.py

```diff
@@ -69,22 +69,13 @@ def is_item_available(item_pid):
     return search.execute().hits.total == 0
 
 
-def is_item_can_request(item_pid):
-    """Return True if the given item can be requested, False otherwise."""
+def can_be_requested(loan):
+    """Return True if the given record can be requested, False otherwise."""
     config = current_app.config
-    cfg_item_can_request = \
-        config["CIRCULATION_POLICIES"]["request"]["can_request"]
-
-    return cfg_item_can_request(item_pid=item_pid)
-
-
-def is_document_can_request(document_pid):
-    """Return True if the given document can be requested, False otherwise."""
-    config = current_app.config
-    cfg_document_can_request = \
-        config["CIRCULATION_POLICIES"]["request"]["can_request"]
-
-    return cfg_document_can_request(document_pid=document_pid)
+    cfg_can_request = config["CIRCULATION_POLICIES"]["request"].get(
```

Comment:
Its not very clear to me why we should go full complexity on our first step.
Instead of the nested `CIRCULATION_POLICIES.request.can_request` we can just have a variable 
`CIRCULATION_CAN_BE_REQUESTED`. If in the future is required to enrich the policies we can expand them but doing it now it feels unnecessary. 

But aside from that, at least I would keep the configuration variable name `can_request` the same as the function name `can_be_requested` to avoid further confusion and be able to find them all with a global search.

Reply:
We have put the `request` configuration in the `CIRCULATION_POLICIES` using the same model as the `checkout`.

Reply:
We changed the can_request key in the configuration to can_be_requested